cypress/e2e/hub/execution-environments.cy.ts

```diff
@@ -1,35 +1,42 @@
+import { hubAPI } from '../../support/formatApiPathForHub';
 import { ExecutionEnvironments } from './constants';
 
-describe('Execution Environments', () => {
+describe.skip('Execution Environments', () => {
```

Comment:
Do we really want to skip this test? It is just making an assertion about the page itself.

Reply:
@jerabekjiri I will merge this one, we can add a follow up PR if necessary.